package.json

```diff
@@ -37,7 +39,8 @@
     "prettier": "^2.8.8",
     "eslint": "^8.41.0",
     "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-prettier": "^4.2.1"
+    "eslint-plugin-prettier": "^4.2.1",
+    "typescript": "^5.1.6"
```

Comment:
Probably we should use "~" instead of "^" and align the TypesScript version with the one used in the Stencil release version.

Reply:
Good point. Turns out the simplest was to remove the dependency in the parent completely and let npm find the best candidate.